src/garage/tf/misc/tensor_utils.py

```diff
@@ -13,6 +13,22 @@ def run(*input_vals):
     return run
 
 
+def broadcast(param, input_var, batch_dim=None):
```

Comment:
perhaps `broadcast_with_batch`? otherwise it's not clear what this is doing different from regular `tf.broadcast_to`

Reply:
Makes sense

Reply:
i don't think you addressed this?